lms/static/js/groups/views/cohort_discussions.js

```diff
@@ -0,0 +1,114 @@
+var edx = edx || {};
+
+(function ($, _, Backbone, gettext, interpolate_text, NotificationModel, NotificationView) {
+    'use strict';
+
+    edx.groups = edx.groups || {};
+
+    edx.groups.CohortDiscussionsView = Backbone.View.extend({
+
+        /**
+         Add/Remove the disabled attribute on given element.
+
+         Args:
+            $element (JQuery element): The element to disable/enable.
+            disable (Bool): The flag to add/remove 'disabled' attribute.
+        **/
+        setDisabled: function($element, disable) {
+            if (disable) {
+                $element.prop('disabled', 'disabled');
+            } else {
+                $element.prop('disabled', false);
+            }
+        },
+
+        /**
+         Returns the cohorted discussions list.
+
+         Args:
+            selector (string): To select the discussion elements whose ids to return.
+
+         Returns:
+            Cohorted discussions list.
+        **/
+        getCohortedDiscussions: function(selector) {
+            var self=this,
+                cohortedDiscussions = [];
+
+            _.each(self.$(selector), function (topic) {
+                cohortedDiscussions.push($(topic).data('id'))
+            });
+
+            return cohortedDiscussions;
+        },
+
+        /**
+         Save the cohortSettings' changed attributes to the server via PATCH method.
+         It shows the error message(s) if any.
+
+         Args:
+            $element (JQuery element): Messages would be shown before this element.
+            fieldData ( dict ) : Data to update on the server.
+        **/
+        saveForm: function ($element, fieldData) {
+            var self = this,
+                cohortSettingsModel = this.cohortSettings,
+                saveOperation = $.Deferred(),
+                showErrorMessage;
+
+            showErrorMessage = function (message, $element) {
+                self.showMessage(message, $element, 'error');
+            };
+            this.removeNotification();
+
+            cohortSettingsModel.save(
+                fieldData, {patch: true, wait: true}
+            ).done(function () {
+                    saveOperation.resolve();
```

Comment:
This indentation is a little off. It should only be four spaces indented from the line before it.